packages/create-osdk-app/src/cli.ts

```diff
@@ -0,0 +1,404 @@
+  const base: Argv<CliArgs> = yargs(hideBin(args))
+    .version(false)
+    .strict()
+    .help()
+    .command(
+      "$0 [project] [--<option>]",
+      "Create a new OSDK application based on framework templates. Information may be provided through arguments to skip interactive prompts.",
+      (yargs) =>
+        yargs
+          .positional("project", {
+            type: "string",
+            describe: "Project name to create",
+          })
+          .option("overwrite", {
+            type: "boolean",
+            describe: "Overwrite project directory if already exists",
+          })
+          .option("template", {
+            type: "string",
+            describe: "Template name to use",
+          })
+          .option("foundry-url", {
+            type: "string",
+            describe: "URL for the Foundry stack",
+          })
+          .option("application-url", {
+            type: "string",
+            describe: "URL the production application will be hosted on",
+          })
+          .option("skip-application-url", {
+            type: "boolean",
+            describe:
+              "Skip filling in URL the production application will be hosted on",
+          })
+          .option("client-id", {
+            type: "string",
+            describe: "OAuth client ID for application",
+          })
+          .option("osdk-package", {
+            type: "string",
+            describe: "OSDK package name for application",
+          })
+          .option("osdk-registry-url", {
+            type: "string",
+            describe: "URL for NPM registry to install OSDK package",
+          }),
+    );
+
+  const parsed: CliArgs = base.parseSync();
+  const project: string = await promptProject(parsed);
+  const overwrite: boolean = await promptOverwrite(parsed, project);
+  const template: Template = await promptTemplate(parsed);
+  const foundryUrl: string = await promptFoundryUrl(parsed);
+  const applicationUrl: string | undefined = await promptApplicationUrl(parsed);
+  const clientId: string = await promptClientId(parsed);
+  const osdkPackage: string = await promptOsdkPackage(parsed);
+  const osdkRegistryUrl: string = await promptOsdkRegistryUrl(parsed);
+
+  consola.log("");
+  consola.start(
+    `Creating project ${green(project)} using template ${green(template.id)}`,
+  );
+
+  const cwd = process.cwd();
+  const root = path.join(cwd, project);
+
+  if (fs.existsSync(root)) {
+    if (overwrite) {
+      consola.info(`Overwriting existing project directory`);
+      fs.rmSync(root, { recursive: true, force: true });
+      fs.mkdirSync(root, { recursive: true });
+    } else {
+      consola.info(`Reusing existing project directory`);
+    }
+  } else {
+    consola.info(`Creating project directory`);
+    fs.mkdirSync(root, { recursive: true });
+  }
+
+  consola.info(`Copying files into project directory`);
+
+  const templateDir = path.resolve(
+    fileURLToPath(import.meta.url),
+    "../..",
```

Comment:
When run in vitest this looks for `create-osdk-app/src/cli.ts` -> `create-osdk-app/template-*`
When run in tsup bundle this looks for `create-osdk-app/build/js/index.mjs` -> `create-osdk-app/build/template-*`